include/dts-functions.sh

```diff
@@ -1572,6 +1572,16 @@ show_footer(){
   else
     echo -ne "${RED}${SSH_OPT_UP}${NORMAL} to launch SSH server  ${NORMAL}"
   fi
+  if [ "${SEND_LOGS_ACTIVE}" == "true" ]; then
+    echo -ne "${RED}${SEND_LOGS_OPT}${NORMAL} to disable sending DTS logs ${NORMAL}"
+  else
+    echo -ne "${RED}${SEND_LOGS_OPT}${NORMAL} to enable sending logs ${NORMAL}"
```

Comment:
```suggestion
    echo -ne "${RED}${SEND_LOGS_OPT}${NORMAL} to disable sending DTS logs ${NORMAL}"
  else
    echo -ne "${RED}${SEND_LOGS_OPT}${NORMAL} to enable sending DTS logs ${NORMAL}"
```

to keep description consistent.

Reply:
https://github.com/Dasharo/dts-scripts/compare/25b98ad5b0fb8c11d4f4485f0f810cc95f12f759..bfa2a3168fd52e0a672d4c9fb157559f33effb1c